src/test/groovy/com/netgrif/application/engine/petrinet/domain/ImporterTest.groovy

```diff
@@ -361,4 +361,18 @@ class ImporterTest {
 
     }
 
+    @Test
+    void importWithInvalidDataRefLayoutTest() {
+        boolean exceptionThrown = false
+
+        try {
+            petriNetService.importPetriNet(new FileInputStream("src/test/resources/invalid_data_ref_layout.xml"), VersionType.MAJOR, superCreator.getLoggedSuper()).getNet()
+        } catch (IllegalArgumentException e) {
```

Comment:
It's better to use exception assertion to check if a function ends up throwing an exception with a specific message or type.
See https://www.baeldung.com/assertj-exception-assertion for more details.